src/frontends/onnx/frontend/src/ops_bridge.cpp

```diff
@@ -573,6 +574,7 @@ OperatorsBridge::OperatorsBridge() {
     REGISTER_OPERATOR_WITH_DOMAIN(MICROSOFT_DOMAIN, "FusedConv", 1, fused_conv);
     REGISTER_OPERATOR_WITH_DOMAIN(MICROSOFT_DOMAIN, "FusedGemm", 1, fusedgemm);
     REGISTER_OPERATOR_WITH_DOMAIN(MICROSOFT_DOMAIN, "EmbedLayerNormalization", 1, embed_layer_normalization);
+    REGISTER_OPERATOR_WITH_DOMAIN(MICROSOFT_DOMAIN, "Range", 1, range);
```

Comment:
As I can see - implementations between existing Range and com.microsoft.Range are same. I suggest to do not add it as a separate implementation, but just reuse existing implementation with com.microsoft.Range as an alias.

Reply:
@gkrivor Did I understand correctly that in the com.microsoft.Range implementation I should use Range implementation? Or while registering I should change the "Range" reference to something else and delete the implementation of the com.microsoft.Range?

Reply:
@vbayeva not exactly. Did you find any difference between existing Range implementation and com.microsoft.Range? I didn't (maybe I'm wrong, re-check!). If there is not difference:
1. I suggest to do not implement it (remove com.microsoft/range.cpp/hpp files).
2. Just leave a line     REGISTER_OPERATOR_WITH_DOMAIN(MICROSOFT_DOMAIN, "Range", 1, range);
3. Keep your tests

If I'm wrong and it has a difference - need to make it clear in source because currently it is same with original Range.

Reply:
Hey @gkrivor, @vbayeva the only minor difference I could find was that `delta` input of [com.microsoft.Range](https://github.com/microsoft/onnxruntime/blob/main/docs/ContribOperators.md#commicrosoftrange) is optional and defaults to 1, whereas `delta` of [Range](https://github.com/onnx/onnx/blob/main/docs/Operators.md#range) is not optional. @gkrivor should we implement this in the source?

Reply:
@siddhant-0707 good catch! Yes, this is significant behavior change. But there is another issue I see in existing approach: we will have two versions of op::set_1::range for ai.onnx and op::set_1::range for com.microsoft...
Let me dig into this direction, I'm refactoring this part of code, so, I will return with an update...